sevntu-checks/src/main/java/com/github/sevntu/checkstyle/checks/coding/Jsr305AnnotationsCheck.java

```diff
@@ -585,22 +586,17 @@ private static boolean isPrimitiveType(DetailAST ast) {
     /**
      * Checks whether token is array or elipsis.
      *
-     * @param identToken
+     * @param typeToken
```

Comment:
Previously named `identToken`, this DetailAST was actually never an IDENT. In every UT we had, this method was passed a `TYPE` ast.